jplag/src/main/java/de/jplag/reporting/reportobject/model/ComparisonReport.java

```diff
@@ -4,58 +4,15 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class ComparisonReport {
+/**
+ * ReportViewer DTO for the comparison of two submissions.
+ * @param firstSubmissionId id of the first submission
+ * @param secondSubmissionId id of the second submission
+ * @param matchPercentage similarity in percent. between 0f and 100f.
+ * @param matches the list of matches found in the comparison of the two submissions
+ */
+public record ComparisonReport(@JsonProperty("first_submission_id") String firstSubmissionId,
```

Comment:
Since these property names appear very very often in the json files, it could even be a good idea to choose shorter names, despite the loss in readability.